### DISCUSSION TITLE
New ui is good but vuetify would have [Nov 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4422] [Discord Threads]
> 🙋‍♂️ **ryzenway**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Braks**  
#### To give a little bit of background info on why we chose Ant over Vuetify for our migration   
####    
#### - When we started migrating, we were in fact using Vuetify but at the time it was in heavy beta state with *a lot* of features/components still missing which caused a lot of blocks   
#### - Vuetify wasn't too clear on *when* they would provide proper Vue3 support so it was not apparent to us if we could wait for a release that would work for us (Vuetifys official Vue3 release was this week I believe)   
#### - Ant was ready to use for Vue3   
#### - It's also open source so I don't see why it being affiliated with China would be *any* issue whatsoever - you can see the code yourself if you're worried the CCP might steal your data 😅  
#  
> 🙋‍♂️ **Russell**  
#### I listened to the Vuetify call and it sounds like it’s still not 100% ready for Vue3 🫤  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1033465216417144862)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Reply by williamstein** [reply]
I really like the look, style and completeness of ant-design, and I'm glad you are using it.
